frontend/packages/dev-console/src/components/import/pipeline/pipeline-template-utils.ts

```diff
@@ -27,7 +27,7 @@ export const createPipelineForImportFlow = async (formData: GitImportFormData) =
   const template = _.cloneDeep(pipeline.template);
 
   template.metadata = {
-    name: `${name}-${template.metadata.name}`,
+    name: `${name}`,
```

Comment:
@rohitkrai03 I assume you tested the scenario about an existing Pipeline with this name? This hopefully errors out like any other resource we create when names duplicate.

Reply:
@andrewballantyne Yes, we get that for every kubernetes resource.

Reply:
I assumed as much - just wanted to make sure... it was an explicit agreement with PM.